Marlin/src/lcd/extui/lib/mks_ui/draw_ui.cpp

```diff
@@ -70,8 +70,8 @@ uint8_t printing_rate_update_flag;
 
 extern bool once_flag;
 extern uint8_t sel_id;
-extern uint8_t public_buf[512];
-extern uint8_t bmp_public_buf[17 * 1024];
+extern uint8_t public_buf[513];
+extern uint8_t bmp_public_buf[14 * 1024];
```

Comment:
What is 14?

Reply:
513 and 14 correctly match the non-extern array sizes.